test/post.test.js

```diff
@@ -93,6 +93,27 @@ describe('Post data class tests', () => {
     expect(() => createPostWithFeed(feed)).not.toThrow();
   });
 
+  test('Post constructor should throw if invalid dates is passed', () => {
+    const createPostWithDates = (datePublished, dateUpdated) =>
+      new Post(data.title, data.html, datePublished, dateUpdated, data.url, data.guid, feed);
+    expect(() =>
+      createPostWithDates(
+        new Date('Thu, 20 Nov 2014 18:59:18 UTC'),
+        new Date('Fri, 28 Nov 2014 18:59:18 UTC')
+      )
+    ).not.toThrow();
+    expect(() =>
+      createPostWithDates(new Date('Thu, 20 Nov 2014 18:59:18 UTC'), '23')
```

Comment:
should we just use 'string' here instead of '23', I want to avoid the situation where someone might miss the `' '` and think this is actually an integer

Reply:
True! I didn't about that, will fix